roles/endpoints/defaults/main.yml

```diff
@@ -287,6 +287,7 @@ endpoints:
     name: barbican
     type: key-management
     description: Key Management Service
+    client_keystone_version: v3
```

Comment:
It gets used in nova.conf/cinder.conf. I left it in there because it seems that the Castellan client currently depends on v3 but I wanted to make it configurable.